Content.Server/_ES/Masks/Traitor/Components/ESAddMaskOnUseComponent.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Content.Shared._ES.Masks;
+using Content.Shared._ES.Masks.Components;
+using Content.Shared.EntityEffects;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server._ES.Masks.Traitor.Components;
```

Comment:
All of this code can pretty easily live in shared. Moving it there will make it feel better for basically no cost. All it needs is popup calls being changed to shared variants.